index.html

```diff
@@ -1101,12 +1101,45 @@ <h2 id="conformance-classes">Conformance Classes</h2>
     </section>
     <section class="normative">
         <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
-        <p>The <a data-cite="VC-DATA-MODEL-2.0#securing-mechanism-specifications"></a> describes
-            the approach taken by JSON Web Tokens to secure JWT Claims Sets as <i>applying an
-                <code>external proof</code></i>.
+        <p>
+            The <a data-cite="VC-DATA-MODEL-2.0#securing-mechanism-specifications"></a> describes
+            the approach taken by this specification to secure JSON and CBOR
+            claims by <i>applying an <code>enveloping proof</code></i>.
         </p>
-        <p>The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">Securing
-            Mechanisms</a> apply to securing
+        <p>
+            This specification defines how to secure different data structures 
+            using various <code>enveloping proof</code> mechanisms:</p>
+        <dl>
+        <dt>JSON Web Tokens (JWTs):</dt>
+        <dd>JWTs secure JSON Web Token Claim Sets. These are JSON objects 
+        containing claims about an entity (typically the subject of the JWT).</dd>
+        <dt>Selective Disclosure JWTs (SD-JWTs): </dt>
+        <dd>SD-JWTs secure JSON Claim Sets, similar to JWTs, but with added
+         selective disclosure capabilities. This allows for parts of the 
+         claim set to be selectively revealed or withheld.</dd>
+         
+        <dt>CBOR Object Signing and Encryption (COSE):</dt>
+        <dd>COSE secures CBOR (Concise Binary Object Representation) data
+        structures. CBOR is a binary data format that is more compact than
+        JSON and is designed for constrained environments.</dd>
+        </dl>
+
+        <p>In the context of Verifiable Credentials:</p>
+        <ul>
+            <li>When using JWTs, the Verifiable Credential or Presentation is
+             encoded as a JSON Web Token Claim Set.</li>
+            <li>When using SD-JWTs, the Verifiable Credential or Presentation 
+            is encoded as a JSON Claim Set with selective disclosure features.</li>
+            <li>When using COSE, the Verifiable Credential or Presentation is 
+            encoded as a CBOR data structure.</li>
+        </ul>
+        <p>
+         In all cases, the underlying data model of the Verifiable Credential
+         or Presentation remains consistent with the [[VC-DATA-MODEL-2.0]],
+         but the encoding and security mechanism differ.</p>
+        <p>
+        The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">
+        Securing Mechanisms</a> apply to securing
             <code>application/vc-ld+jwt</code> and
             <code>application/vp-ld+jwt</code>,
             <code>application/vc-ld+sd-jwt</code> and
```

Comment:
The examples now use `application/vc-ld+sd` and `application/vp-ld+sd`. These should also be what is in the document, including the lines following the selection above, which are out of range of selection.
```suggestion
            <code>application/vc+jwt</code> and
            <code>application/vp+jwt</code>,
            <code>application/vc+sd-jwt</code> and
```
plus
```
            <code>application/vp+sd-jwt</code>,
            as well as
            <code>application/vc+cose</code> and
            <code>application/vp+cose</code>.
```

Reply:
need to revert this. the media types in main still use `-ld` https://w3c.github.io/vc-jose-cose/#media-types

this should be resolved after TPAC and #282